docs/stardoc/scala_with_scalafmt.md

```diff
@@ -384,6 +396,14 @@ scala_test(<a href="#scala_test-name">name</a>, <a href="#scala_test-config">con
 </pre>
 
 
+Compiles and links a collection of Scala tests.
+
+To buid and run all tests: `bazel test <target>`
+
+To build and run a specific test: `bazel test <target> --test_filter=<test>`
```

Comment:
Do we cover the `--test_filter=<text>` syntax anywhere? If not, including a section on what the filter expects would be very helpful. It is my understanding that `test_filter` syntax is unique per ruleset https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_filter

Reply:
Ah I didn't know this varied by framework. I'll mention that this varies by framework and add the syntax for specs2